painter/src/root_painter.py

```diff
@@ -348,7 +348,7 @@ def update_class(self, class_name):
     def get_seg_path(self, fname=None):
         if fname is None:
             fname = self.fname
-        seg_fname = fname.replace('.nrrd', '.nii.gz')
+        seg_fname = fname.replace('.nrrd', '.nii.gz')  # @TODO: How to handle this with .nii files?
```

Comment:
segmentations are generated by RootPainter3D so I don't think we need to handle anything other than .nii.gz